webapp/channels/src/selectors/posts.ts

```diff
@@ -52,3 +58,69 @@ export function isInlineImageVisible(state: GlobalState, postId: string, imageKe
 
     return getGlobalItem(state, StoragePrefixes.INLINE_IMAGE_VISIBLE + currentUserId + '_' + postId + '_' + imageKey, !imageCollapsed);
 }
+
+export function makeCanWrangler() {
+    return createSelector(
```

Comment:
Do we need to check the feature flag here as well?

Reply:
It's checked in the `moveThreadsEnabled` selector argument, the `enabled` boolean will reflect the proper value